config/m4/compiler.m4

```diff
@@ -17,8 +17,20 @@ AC_ARG_ENABLE(debug,
         [],
         [enable_debug=no])
 AS_IF([test "x$enable_debug" == xyes],
-        [BASE_CFLAGS="-O0 -D_DEBUG $BASE_CFLAGS"],
-        [BASE_CFLAGS="-O3 $BASE_CFLAGS"])
+        [BASE_CFLAGS="-D_DEBUG $BASE_CFLAGS"],
+        [])
```

Comment:
enable_debug should be mutual exclusive with enable_fast and probably set enable_fast=0
or add check in enable_fast that enable_debug is not set

Reply:
ok, done